examples/platform/silabs/uart.cpp

```diff
@@ -195,7 +199,7 @@ static Fifo_t sReceiveFifo;
 #if SLI_SI91X_MCU_INTERFACE == 0
 static void UART_rx_callback(UARTDRV_Handle_t handle, Ecode_t transferStatus, uint8_t * data, UARTDRV_Count_t transferCount);
 #endif // SLI_SI91X_MCU_INTERFACE == 0
-static void uartSendBytes(uint8_t * buffer, uint16_t nbOfBytes);
+static void uartSendBytes(UartTxStruct_t * bufferStruct, uint16_t nbOfBytes);
```

Comment:
since UartTxStruct_t has the length already we could drop `nbOfBytes` param

Reply:
True, somehow I thought length was size when I edited that